code/client/src/sdk/entities/c_car.cpp

```diff
@@ -83,6 +75,20 @@ namespace SDK {
         return 0.0f;
     }
 
+    void C_Car::SetActualFuel(float fuel) {
+        hook::this_call<void>(gPatterns.C_Car__SetActualFuel, this, fuel);
+
+        // TODO: fix me, doesn't work yet
+        // uint64_t C_Motor = *(uint64_t *)(*(uint64_t *)((uint64_t)this + C_Motor_Offset) + 0x8);
+        // if (C_Motor) {
+        //     hook::this_call(gPatterns.C_Motor__SetFuel, C_Motor, fuel);
+        // }
```

Comment:
Either fix or don't commit temporary comments

Reply:
This was the original code that didn't work, ok to remove it?

Reply:
Yes